docs/tools/erc725js/providers.md

```diff
@@ -8,22 +8,38 @@ The provider by which `@erc725/erc725.js` will request blockchain data is set on
 
 The following provider types are supported:
 
-## Web3
+## RPC URL
 
-The following code snippet will use the web3 provider available at web3.providers from the corresponding `web3` library.
+An RPC URL can be passed to the configuration object.
```

Comment:
@Hugoo this config object I was referring to. Is this is incorrect we should change it. Also we should add examples as its not clear where the provider is passed.

Reply:
I still think we should have examples here otherwise we provide no context of how the provider is used

Reply:
Yeah i get your point - the current provider page is very bad. It misses a line that shows how to use it.
@skimaharvey the screenshot you shared with me was fine:

<img width="1020" alt="image" src="https://user-images.githubusercontent.com/477945/194346630-ecd177dd-4578-42ed-9b62-f499faeb13f1.png">

Let's simply not import the schema - as this is not the focus - and put `[]` instead for the schema.
Also for the up address, just use `0x...`